src/LandIce/CMakeLists.txt

```diff
@@ -8,7 +8,6 @@
 
 set(SOURCES
   evaluators/LandIce_BasalFrictionCoefficient.cpp
-  evaluators/LandIce_BasalFrictionCoefficientGradient.cpp
```

Comment:
This evaluator was used for the regularization term inside SurfaceVelocityMismatch. Are we sure it's not needed anymore? (Note: I'm happy if it can be removed, just double checking).